frontend/src/components/Footer/Footer.js

```diff
@@ -15,7 +15,7 @@ export const Footer = props => {
     return (
         <footer className={ classes.root }>
             <Paragraph>
-                This application is developed by the <a href="http://txscience.renci.org/" target="_blank" rel="noopener noreferrer">Translational Science Team at RENCI</a>.
+                This application is developed at RENCI.
```

Comment:
i think i agree with @hyi on this. 

however, i will say that this may need to be addressed at some point because Translational Science Team was kind of a Kimberly thing.

Reply:
fixed. Thanks!